crates/cranelift/src/func_environ.rs

```diff
@@ -1317,11 +1317,12 @@ impl<'module_environment> cranelift_wasm::FuncEnvironment for FuncEnvironment<'m
         mut pos: cranelift_codegen::cursor::FuncCursor<'_>,
         index: cranelift_wasm::GlobalIndex,
     ) -> WasmResult<ir::Value> {
-        debug_assert_eq!(
-            self.module.globals[index].wasm_ty,
-            WasmType::ExternRef,
-            "We only use GlobalVariable::Custom for externref"
-        );
+        // TODO(dhil) fixme: need to check that wasm_ty is indeed a Ref.
```

Comment:
I don't know anything about GlobalVariable::Custom, but if the message is true we can probably leave this as is but with EXTERN_REF